client/src/components/Billing/index.js

```diff
@@ -1,104 +1,103 @@
-import React, {Component} from 'react';
-import PropTypes from 'prop-types';
-import StarIcon from '@material-ui/icons/StarBorder';
-import {withStyles} from '@material-ui/core/styles';
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import StarIcon from "@material-ui/icons/StarBorder";
+import { withStyles } from "@material-ui/core/styles";
 import {
   Card,
   CardContent,
   CardHeader,
   CssBaseline,
   Grid,
   Typography,
-  Button,
-} from '@material-ui/core';
-import {Link} from 'react-router-dom';
+  Button
+} from "@material-ui/core";
+import { Link } from "react-router-dom";
 
-import * as ROUTES from '../../constants/routes';
-import Checkout from '../Stripe/checkout';
+import * as ROUTES from "../../constants/routes";
+import Checkout from "../Stripe/checkout";
 // import {fire} from '../Auth/firebaseConfig';
 
 const styles = theme => ({
-  '@global': {
+  "@global": {
     body: {
-      backgroundColor: theme.palette.common.white,
-    },
+      backgroundColor: theme.palette.common.white
+    }
   },
   layout: {
-    width: 'auto',
+    width: "auto",
     marginLeft: theme.spacing.unit * 3,
     marginRight: theme.spacing.unit * 3,
     [theme.breakpoints.up(900 + theme.spacing.unit * 3 * 2)]: {
       width: 900,
-      marginLeft: 'auto',
-      marginRight: 'auto',
-    },
+      marginLeft: "auto",
+      marginRight: "auto"
+    }
   },
   heroContent: {
     maxWidth: 600,
-    margin: '0 auto',
-    padding: `${theme.spacing.unit * 8}px 0 ${theme.spacing.unit * 6}px`,
+    margin: "0 auto",
+    padding: `${theme.spacing.unit * 8}px 0 ${theme.spacing.unit * 6}px`
   },
   cardHeader: {
-    backgroundColor: theme.palette.grey[200],
+    backgroundColor: theme.palette.grey[200]
   },
   cardPricing: {
-    display: 'flex',
-    justifyContent: 'center',
-    alignItems: 'baseline',
-    marginBottom: theme.spacing.unit * 2,
+    display: "flex",
+    justifyContent: "center",
+    alignItems: "baseline",
+    marginBottom: theme.spacing.unit * 2
   },
   cardActions: {
-    [theme.breakpoints.up('sm')]: {
-      paddingBottom: theme.spacing.unit * 2,
-    },
-  },
+    [theme.breakpoints.up("sm")]: {
+      paddingBottom: theme.spacing.unit * 2
+    }
+  }
 });
 
 /* Filled in wording & pricing for now.  I expect we will decide on 
 exact wording later on */
 
 const tiers = [
   {
-    title: 'Newbie',
+    title: "Newbie",
```

Comment:
Just a suggestion you may want to move this out into it's own file so you can lessen you componets file.

Reply:
Maybe later.  I don't want to mess with it right now.